helpers/did/did.go

```diff
@@ -43,5 +43,13 @@ func CreateDidDocument(kps *crypto.KeyPairs) *model.DidDocument {
 				PublicKeyMultibase: kps.AssertionMethodPublicKey,
 			},
 		},
+		KeyAgreement: []model.KeyMaterial{
```

Comment:
any tests we can add?

Reply:
sure we can test that the key agreement is being added to the generated did